### ISSUE TITLE
Error parsing Unreal Notation file

### ISSUE THREAD

**Issue by ghost** [Square789/BL2_skingen#17]
When trying to generate a skin from the extracted files, the generator always throws the following error:

C:\BL2-Ext>skingen -in CD_Assassin_Skin_AsPower_SF -o C:\bl2conv
INFO     | run                 : Input directory: C:\BL2-Ext\CD_Assassin_Skin_AsPower_SF
INFO     | run                 : Output directory: C:\bl2conv
INFO     | run                 : Seeking for props files...
INFO     | run                 : Parsing props files and getting textures...
CRITICAL | _parse_props_files  : Error parsing Unreal Notation file: Index-less list key at around 3933

Game files were extracted with up to date UModel (downloaded as of may-16), running Python 3.8.10 x64 and generator version 1.3.0.

Tried other versions for both the generator and Python, also other settings when exporting, nothing worked. Specifying other skins changes the number at the end of the error.

**Comment by Square789** [comment]
Hey, sorry for taking so long.
There seems to have been a change in the output files and this hastily cobbled together parser fails to handle a newly appeared empty list. I've fixed that, but the output of the parser on this specific skin (If it is "A Cleansing Fire") looks nowhere near the indended result.
Luckily for you this is precisely the one skin that I needed to manually recreate in the process I wrote this tool for, I attached the files to this comment if they're still relevant after 2 weeks: [CD_Assassin_Skin_AsPower_SF.zip](https://github.com/Square789/BL2_skingen/files/6567134/CD_Assassin_Skin_AsPower_SF.zip)
